.github/workflows/ci.yml

```diff
@@ -65,3 +65,15 @@ jobs:
           -Psigning.keyId="${{ secrets.SIGNING_KEY_ID }}"
           -Psigning.password="${{ secrets.SIGNING_PASSWORD }}"
           -Psigning.secretKeyRingFile="$HOME/secring.gpg"
+
+      - name: PublishDocumentation
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          ./gradlew dokkaHtml
+          git config user.email "ci-build@juul.com"
+          git config user.name "ci-build"
+          git checkout --orphan gh-pages
+          git rm -rf .
+          git add ./core/build/dokka/html/core/
```

Comment:
Do we need to move these files before adding them? So that the starting `index.html` is at the root?

Reply:
Good question. Honestly, I'm not exactly sure without seeing what Github pages does when this happens.

Reply:
Looks like it hosts whatever is in the `gh-pages` branch.

The [current `gh-pages`](https://github.com/JuulLabs/kable/tree/gh-pages) just has an `index.html` file in it and [renders as excepted](https://juullabs.github.io/kable/).

Reply:
Got it working on my fork.